__tests__/app.test.js

```diff
@@ -22,3 +23,32 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topic objects", () => {
+    const topics = [
+      {
+        description: "The man, the Mitch, the legend",
+        slug: "mitch",
+      },
+      {
+        description: "Not dogs",
+        slug: "cats",
+      },
+      {
+        description: "what books are made of",
+        slug: "paper",
+      },
+    ];
+
```

Comment:
The data above does not need to be hardcoded, what tools are avialable to us so we can assert that everything is as it should be? for instance if your test data had 50 topics this would be a rather lengthy test suite if we declared a variable in each test this way.